_pages/wb2ifs.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: wb2ifs
+pageUrl: https://wb2ifs.github.io/githubtestrepo.github.io/
```

Comment:
@wb2ifs leave out the `https://` and it should work. If you rename your repo to `wb2ifs.github.io`, you'll also notice that the url changes to simply that.